server/index.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const db = require('../db/index.js');
+const bodyParser = require('body-parser');
+// for cross-origin-request
+const cors = require('cors');
+const path = require('path');
+
+let app = express();
+app.use(bodyParser.json());
+app.use(cors());
+
+//server static content
+app.use(express.static(path.join(__dirname, '../client/dist')))
+app.get('/reviews', function(req, res) {
+    var pageIndex = Number(req.url.split('=')[1]);
```

Comment:
try to switch these out for lets/const! 

Also, how will you know which id is the room you're looking for reviews for? will that be included in this get url? For example, my get request looks like '/rooms/:listingId/similar_listings', where :listingId will tell me which room it is